app/Command/Publish.hs

```diff
@@ -13,6 +13,11 @@ import           Language.PureScript.Publish.ErrorsWarnings
 import           Options.Applicative (Parser)
 import qualified Options.Applicative as Opts
 
+manifestPath :: Parser FilePath
```

Comment:
I didn't really think much about it yet, and we can definitely change it. I don't feel strongly either way.